pvnet_app/data.py

```diff
@@ -1,30 +1,59 @@
+import pandas as pd
 import xarray as xr
 import xesmf as xe
 import logging
 import os
 import fsspec
+from datetime import timedelta
+import ocf_blosc2
 
-from pvnet_app.consts import sat_path, sat_15_path, nwp_path
+from pvnet_app.consts import sat_path, nwp_path
 
 logger = logging.getLogger(__name__)
 
 this_dir = os.path.dirname(os.path.abspath(__file__))
 
+sat_5_path = "sat_5_min.zarr"
+sat_15_path = "sat_15_min.zarr"
+
+
 def download_sat_data():
     """Download the sat data"""
     fs = fsspec.open(os.environ["SATELLITE_ZARR_PATH"]).fs
-    fs.get(os.environ["SATELLITE_ZARR_PATH"], "sat.zarr.zip")
-    os.system(f"rm -r {sat_path}")
-    os.system(f"unzip sat.zarr.zip -d {sat_path}")
+    fs.get(os.environ["SATELLITE_ZARR_PATH"], "sat_5_min.zarr.zip")
+    os.system(f"rm -r {sat_5_path} {sat_15_path} {sat_path}")
+    os.system(f"unzip sat_5_min.zarr.zip -d {sat_5_path}")
     
     # Also download 15-minute satellite if it exists
     sat_latest_15 = os.environ["SATELLITE_ZARR_PATH"].replace("sat.zarr", "sat_15.zarr")
     if fs.exists(sat_latest_15):
         logger.info("Downloading 15-minute satellite data")
-        fs.get(sat_latest_15, "sat_15.zarr")
-        os.system(f"unzip sat_15.zarr.zip -d {sat_15_path}")
+        fs.get(sat_latest_15, sat_15_path)
+        os.system(f"unzip sat_15_min.zarr.zip -d {sat_15_path}")
         
 
+def preprocess_sat_data(t0):
```

Comment:
worth adding `sat_5_path` as a input here, with the default being above. Then it will be easier to test. Perhaps the same for `sat_15_path`

Reply:
Got this working without that